inference-engine/src/vpu/myriad_plugin/myriad_config.h

```diff
@@ -34,7 +34,7 @@ VPU_DECLARE_ENUM(MovidiusDdrType,
     MICRON_1GB  = 4,
 )
 
-class MyriadConfig final : public ParsedConfig {
+class MyriadConfig : public virtual ParsedConfig {
```

Comment:
Do we really need virtual inheritance?

Reply:
You can see explanation in commit message. Please read it and if there is something unclear to you I'll answer your concern here.